packages/babel-make-styles/src/plugin.ts

```diff
@@ -408,6 +415,9 @@ export const plugin = declare<Partial<BabelPluginOptions>, PluginObj<BabelPlugin
   api.assertVersion(7);
 
   const pluginOptions: BabelPluginOptions = {
+    babelOptions: {
+      presets: ['@babel/preset-typescript'],
```

Comment:
We agree to leave `@babel/preset-typescript` as a default one, we can reconsider this later.